src/public/app/widgets/dialogs/revisions.ts

```diff
@@ -101,9 +102,9 @@ export default class RevisionsDialog extends BasicWidget {
     private revisionId: string | null;
 
     //@ts-ignore
```

Comment:
We need to remove the `//@ts-ignore`. Please check all the other files in `src/public/app` just to make sure.

Reply:
done, thanks for noticing :-)
now we don't have any bootstrap related ts-ignores anymore – remaining ts-ignores are for other unrelated issues